fe/model_rabfe.py

```diff
@@ -285,6 +284,14 @@ def _predict_a_to_b(
             seed
         )
 
+        # uncomment if we want to visualize.
+        combined_topology = model_utils.generate_imaged_topology(
```

Comment:
Note that this does rely on the image_molecules() call inside mdtraj, which has an extremely small, but non-zero, possibility of segfaulting. Not sure if this is truly safe or not.